src/lib/components/Nav.svelte

```diff
@@ -18,29 +18,29 @@
     open = false
   }
   const pages = [
-    { name: 'Home', href: '#home' },
+    { name: 'HOME', href: '#home' },
```

Comment:
No change necessary, but preferable to keep data always in one formatting convention, then using CSS with `uppercase`, `capitalize`, or `lowercase` for letter case changes.

Reply:
ok